### PR TITLE
doc for required modules

### PR DESCRIPTION
**Pull request by mmatera**
This contains the docstring fixes in modules used in https://github.com/Mathics3/mathics-core/tree/ByteArray-box (see comment https://github.com/Mathics3/mathics-core/pull/669#issuecomment-1362618569)

### CODE REVIEW

File: mathics/builtin/drawing/image.py [hunk 1]
```diff
@@ -146,6 +148,8 @@ def extract(im, evaluation):
 
 class ImageImport(_ImageBuiltin):
     """
+    <url>:WMA link:https://reference.wolfram.com/language/ref/ImageImport.html</url>
```
Comment (rocky) [inline mathics/builtin/drawing/image.py:151]: There is a bug in the doc handling that lowercases interior I's so this link turns out to be invalid. 

I guess we will handle that in another PR.

**Comment by rocky** [comment]
If this passes CI (and I don't know why it shouldn't. I did test this locally), then let's merge this. 

After that I will rebase #669 and put in a PR for that.

I will also try rebasing the other PRs.

File: mathics/builtin/drawing/image_internals.py [hunk 2]
```diff
@@ -63,7 +63,7 @@ def pixels_as_float(pixels):
         return pixels.astype(numpy.float32) / 255.0
     elif dtype == numpy.uint16:
         return pixels.astype(numpy.float32) / 65535.0
-    elif dtype == numpy.bool:
+    elif dtype == bool:
```
Comment (rocky) [inline mathics/builtin/drawing/image_internals.py:66]: BTW this entire module should be moved to `mathics.eval`.

Comment (mmatera) [reply mathics/builtin/drawing/image_internals.py:66]: BTW: shouldn't be ```dtype is bool``` ?

Comment (rocky) [reply mathics/builtin/drawing/image_internals.py:66]: Yes that is correct. I have made the change in b0a324fb

**Pull request merged**